index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>UMD Test</title>
```

Comment:
Is this file here on purpose?

Reply:
Yes, you can run `yarn test:umd` and it'll bring up a simple http-server. Initially place it inside the test folder, however, it couldn't load the JS from one level up in the folder structure.